doc/orchestrator.rst

```diff
@@ -497,12 +507,12 @@ to the ``create_model()`` function and assign to the variable ``model``.
 
 Step 2: Colocate
 """"""""""""""""
-To colocate the model, use the ``Model.colocate_db_tcp()`` function to
+To colocate the model, use the ``Model.colocate_db_uds()`` function to
 Colocate an Orchestrator instance with this Model over TCP/IP.
```

Comment:
UDS isn't a TCP/IP connection, but a Unix domain socket connection

Reply:
note that I need to watch a vid on this